src/services/obs/backend.rs

```diff
@@ -441,10 +500,28 @@ impl Backend {
             .map_err(|e| new_request_send_error("read", path, e))
     }
 
-    pub(crate) async fn get_head_object(
+    pub(crate) fn put_object_request(
         &self,
         path: &str,
-    ) -> Result<isahc::Response<isahc::AsyncBody>> {
+        body: AsyncBody,
+    ) -> Result<Request<AsyncBody>> {
+        let url = format!("{}/{}", self.endpoint, percent_encode_path(path));
+
+        let mut req = isahc::Request::put(&url);
+
+        if let Some(content_length) = body.len() {
+            req = req.header(CONTENT_LENGTH, content_length)
+        }
+
+        // Set body
```

Comment:
We don't need comments for trivial operations.

Reply:
Fixed